package.json

```diff
@@ -10,15 +10,15 @@
   },
```

Comment:
- [ ] **Optional:** Kindly use the JSON script key `start` as the command to run your app instead of `dev`. [This is conventionally used among devs.](https://docs.npmjs.com/cli/v8/commands/npm-start#synopsis)